### ISSUE TITLE
[UX] CKEditor: denote that right-click is required to edit tables

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4065]
From #1251:

@BWPanda:
> I just tested on a node and I was able to edit the width successfully (after taking ages to work out I had to right-click in the table to edit it).

**Comment by BWPanda** [comment]
Yeah, this really isn't obvious. I spent ages clicking in the table, then clicking the Table button in the editor (thinking I was editing the selected table) but it kept inserting a new table into the existing one. From what I could see the only way to edit a table's properties is to right-click and select the option that way (unlike every other editor button (e.g. click in a link, then click the Link button to edit it)).

**Comment by klonos** [comment]
I think that at some point you could do that by double-clicking ...but could be thinking a different WYSIWYG.
